### DISCUSSION TITLE
is there anyway to speed up booting

### DISCUSSION THREAD

**Discussion by X7md** [Darin755/browser-linux#8] [Q&A] [answered]
Booting taking about 1 min, but at copy.sh/v86 buildroot image take less then that...

anything can speed it up?

**Comment by Darin755** [comment, marked as answer]
There a careful balancing act between functionality and speed. I tried to get the best of both.

You can wait for it to boot and then select 'save' in the menu. This will save a snapshot in cache that will load on a refresh.

**Comment by Darin755** [comment]
Hey,
I just wanted to let you know that I'm working on a way to restore snapshots on boot to allow for instant loading. I isn't quiet ready yet but I hope that this will speed up load times immensely. 

In the mean time I have added some kernel optimizations to try to speed some things up.

Edit:
I have a working version [here](https://darin755.github.io/browser-linux?initial=minimal.bin.zst) you want to try it
